src/Spryker/Zed/AppKernel/Business/Writer/ConfigWriter.php

```diff
@@ -59,14 +59,14 @@ public function saveConfig(AppConfigTransfer $appConfigTransfer): AppConfigRespo
             return $this->getSuccessResponse($appConfigTransfer);
         } catch (Throwable $throwable) {
             $this->getLogger()->error(
-                static::FAILED_TO_REGISTER_TENANT_MESSAGE,
+                static::FAILED_TO_OPERATION_TENANT_MESSAGE,
```

Comment:
why is the message is generic here if the method is `saveConfig()`?  
So "Failed to save tenant configuration." is more suitable here

Reply:
it processes connect, new, disconnect statuses. you can check `doSaveAppConfig`